src/frontend/athenaeum/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "athenaeum",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.5.0",
+    "@testing-library/user-event": "^7.2.1",
+    "eslint": "^7.0.0",
+    "eslint-plugin-react": "^7.19.0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-scripts": "3.4.1"
```

Comment:
Let's also add Prettier (unless I missed it) using `npm install --save-dev --save-exact prettier` so it's a dev dependency.